zproject/urls.py

```diff
@@ -322,6 +323,9 @@
     # Endpoints for syncing drafts.
     rest_path("drafts", GET=fetch_drafts, POST=create_drafts),
     rest_path("drafts/<int:draft_id>", PATCH=edit_draft, DELETE=delete_draft),
+    # New scheduled messages are created via send_message_backend.
+    rest_path("scheduled_messages", GET=fetch_scheduled_messages),
+    rest_path("scheduled_messages/<int:scheduled_message_id>", DELETE=delete_scheduled_messages),
```

Comment:
I think this URL format is reasonable to me, but can you please post in "api design" with these details, also mentioning that sending later uses the main "send message" endpoint. I think it's not unlikely that folks will be interested in making `POST /scheduled_messages` be the way to schedule messages rather than the current `POST /messages`, so worth discussing.